client/src/lobby/keyboardInit.ts

```diff
@@ -30,8 +30,8 @@ export default function keyboardInit(): void {
       if (globals.currentScreen === Screen.Lobby) {
         $(".lobby-games-join-first-table-button").click();
       }
-    } else if (event.altKey && (event.key === "c" || event.key === "ç")) {
-      // Alt + c
+    } else if (event.altKey && (event.key === "n" || event.key === "ç")) {
```

Comment:
and the c cedille?

Reply:
ñ

Reply:
Not sure what the second key is for; my guess was that it is mac specific. That's why I need a fix on line 33.

Reply:
Yeah it's mac specific https://www.webnots.com/option-or-alt-key-shortcuts-to-insert-symbols-in-mac-os-x/ so replace the ç with ˜ i guess